### ISSUE TITLE
Adding / Modifying Wrong Config [Solved - User Error]

### ISSUE THREAD

**Issue by 89gsc** [cbaerike/MySQL-Backup-Manager#9]
Hi same issue as I had before, if I add a database with the wrong details (I did this by mistake) and then modify it (i can now see values to modify which is cool) and I know these details are correct it still fails to backup with an error which suggests its user / pass related:

2015-06-29 14:51 - ERROR - Unknown exception occurred during backup: System.IO.IOException: Logon failure: unknown user name or bad password.

   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.Directory.InternalCreateDirectory(String fullPath, String path, Object dirSecurityObj, Boolean checkHost)
   at System.IO.Directory.InternalCreateDirectoryHelper(String path, Boolean checkHost)
   at System.IO.Directory.CreateDirectory(String path)
   at MySQLBackup.Application.Backup.MySQLDumpProcess.WriteBackupFile(String hostName, String databaseName, String output)
   at MySQLBackup.Application.Backup.MySQLDumpProcess.ProcessMySqlDump(Guid databaseId)

I have readded this 3 times now using details I know for certain are correct and have tested in other application and I always get this error, I have this issue on 2 other databases which I entered incorrectly the first time and can not readd them using the correct details they always fail.

**Comment by cbaerike** [comment]
Thanks for the stack trace! The error does not occur when logging in to the database - your MySQL credentials are fine. Instead, the error occurs when the backup is written to disk, specifically when the backup folder is being created (**System.IO.Directory.CreateDirectory(String path)**).

Can you check whether the backup manager account has sufficient permissions to create the folder? Also, what's the total length of the backup folder (including hostname and database name)? This error message may be a red herring, and the actual problem may be a backup path that's longer than 255 characters.

**Comment by 89gsc** [comment]
A thousand apologies! 

I realize now what I did wrong, I set this application up on our small business server while logged in as the admin user, now I am storing all backups on our terrastation which is not part of the domain and has its own admin linux user, there is no issues for myself accessing the terrastation because theres no user called admin but the terrastation kept thinking the windows admin account was the linux admin account on the terrastation so therefor failing.

Anyway I ran the app as myself instead and it works fine, sorry for wasting your time but I really appreciate the help.
